api.go

```diff
@@ -27,6 +27,9 @@ type FileMatch struct {
 	// Ranking; the higher, the better.
 	Score float64 // TODO - hide this field?
 
+	// SubRepositoryScore holds the priority of the repo, if one exists.
```

Comment:
```suggestion
	// RepositoryPriority holds the priority of the repo, if one exists.
```

Reply:
What's the reason for not using the `Score` field?